nova_api.json

```
@@ -0,0 +1,232 @@
+{
+    "label":
+    {
+        "zh-cn": "节点名称(key, display_name)",
+        "ja": "ノード名"
+    },
+    "jump_to":
+    {
+        "zh-cn": "设置跳转(dest)",
+        "ja": "指定ノードへ"
+    },
+    "branch":
+    {
+        "zh-cn": "设置剧情分支(branches)",
+        "ja": "ノードの枝を設置する"
+    },
+    "is_chapter":
+    {
+        "zh-cn": "设置为章节:跳到上一个/下一个章节时会在这个节点停止，但是这个节点不一定显示在章节选择界面",
+        "ja": "ノードを設置する"
```

```suggestion
        "ja": "チャプターを設置する"
```
日文里面应该也要区分“节点”（node）和“章节”（chapter）这两个概念